python/lsst/ctrl/bps/parsl/sites/princeton.py

```diff
@@ -26,6 +26,7 @@ class Tiger(Slurm):
     - ``walltime`` (`str`): time limit for each Slurm job.
     - ``mem_per_node`` (`int`): memory per node (GB) for each Slurm job.
     - ``max_blocks`` (`int`): maximum number of blocks (Slurm jobs) to use.
+    - ``cmd_timeout`` (`int`): timeout (seconds) to wait for a scheduler.
```

Comment:
`timeout (seconds) to wait for Slurm commands.`

Why is it an `int` rather than `float`?

Reply:
The [cmd_timeout](https://github.com/Parsl/parsl/blob/6eae3547f46dcdb3a27e120b559bcbcad8c12ef2/parsl/providers/slurm/slurm.py#L101) argument in the Parsl module is currently configured to accept `int`s. I'm not sure we can get away with requesting a float time here in that case.